### PR TITLE
removed all references to Finicity

### LINKED ISSUE #146: Remove all references to Finicity
There's a good bit of code and database schema that references Finicity.

We need to remove all of that as we won't be using Finicity in any form.

### PR DESCRIPTION
**Pull request by lorenzopalaia**
fix #146

### CODE REVIEW

**Comment by Shpigford** [comment]
Is a migration needed to make the the database changes?

**Comment by lorenzopalaia** [comment]
I'm not entirely sure. Some queries and models referred to Finicity so I guess this is necessary.

**Comment by Shpigford** [comment]
I didn't see any migrations, just changes to the schema. So asking if we should add migrations to make those changes. Unless I'm misunderstanding something? (Possible)

File: apps/server/src/app/app.ts [hunk 1]
```diff
@@ -44,7 +43,6 @@ import {
     toolsRouter,
     publicRouter,
     e2eRouter,
-    adminRouter,
```
Comment (tmyracle) [inline apps/server/src/app/app.ts:47]: This should not be removed. It was just added back for queue service debugging

File: apps/server/src/app/routes/admin.router.ts
Comment (tmyracle) [inline apps/server/src/app/routes/admin.router.ts:1]: Add this file back please

File: apps/server/src/app/routes/index.ts [hunk 2]
```diff
@@ -15,4 +14,3 @@ export { default as plansRouter } from './plans.router'
 export { default as toolsRouter } from './tools.router'
 export { default as publicRouter } from './public.router'
 export { default as e2eRouter } from './e2e.router'
-export { default as adminRouter } from './admin.router'
```
Comment (tmyracle) [inline apps/server/src/app/routes/index.ts:18]: Add back

File: apps/workers/src/main.ts [hunk 3]
```diff
@@ -16,7 +16,6 @@ import {
     workerErrorHandlerService,
 } from './app/lib/di'
 import env from './env'
-import { cleanUpOutdatedJobs } from './utils'
```
Comment (tmyracle) [inline apps/workers/src/main.ts:19]: add back, these all got removed when resolving merge conflicts I'm imagining

File: apps/workers/src/main.ts [hunk 4]
```diff
@@ -127,16 +120,6 @@ syncInstitutionQueue.add(
     {},
     {
         repeat: { cron: '0 */24 * * *' }, // Run every 24 hours
-        jobId: Date.now().toString(),
-    }
-)
-
-syncInstitutionQueue.add(
```
Comment (tmyracle) [inline apps/workers/src/main.ts:134]: This is the only thing that should be changed on this file

File: apps/server/src/app/app.ts (hunk 1 above)
Comment (lorenzopalaia) [reply apps/server/src/app/app.ts:47]: Yes, I noticed it but I thought I was a few commits behind. I'll add everything back.

Comment (tmyracle) [reply apps/server/src/app/app.ts:47]: Thanks! Gotta love merge conflicts 😄

File: apps/workers/src/main.ts (hunk 3 above)
Comment (lorenzopalaia) [reply apps/workers/src/main.ts:19]: Strange, git did everything automatically without asking me for confirmation for conflicts in the merge but I think this is certainly the reason.

**Comment by lorenzopalaia** [comment]
Yeah you're right, I'haven't done any migration.

> I didn't see any migrations, just changes to the schema. So asking if we should add migrations to make those changes. Unless I'm misunderstanding something? (Possible)

You're right, I haven't done any migration.

File: libs/server/features/src/account-connection/account-connection.service.ts [hunk 5]
```diff
@@ -156,9 +156,7 @@ export class AccountConnectionService implements IAccountConnectionService {
             `Disconnected connection id=${connection.id} type=${
                 connection.type
             } provider_connection_id=${
-                connection.type === 'plaid'
-                    ? connection.plaidItemId
-                    : connection.finicityInstitutionId
+                connection.plaidItemId
```
Comment (tmyracle) [inline libs/server/features/src/account-connection/account-connection.service.ts:159]: I missed this in my teller implementation, but I think we can just update these all to connection.tellerEnrollmentId, we shouldn't have any plaid or finicity connections moving forward.

**Comment by lorenzopalaia** [comment]
@tmyracle Fixed previous merge issues. Is there any missing line/file?

File: apps/server/src/app/app.ts [hunk 6]
```diff
@@ -94,7 +93,6 @@ app.use(cors({ origin, credentials: true }))
 app.options('*', cors() as RequestHandler)
 
 app.set('view engine', 'ejs').set('views', __dirname + '/app/admin/views')
-app.use('/admin', adminRouter)
```
Comment (tmyracle) [inline apps/server/src/app/app.ts:97]: this also

File: apps/workers/src/main.ts
Comment (tmyracle) [inline apps/workers/src/main.ts:1]: everything here needs to be added back except for the sync-finicity-institutions related items

File: apps/workers/src/utils.ts
Comment (tmyracle) [inline apps/workers/src/utils.ts:1]: add back

File: libs/server/shared/src/services/queue.service.ts [hunk 7]
```diff
@@ -1,6 +1,6 @@
 import type { AccountConnection, User, Account } from '@prisma/client'
 import type { Logger } from 'winston'
-import type { Job, JobOptions, JobInformation } from 'bull'
```
Comment (tmyracle) [inline libs/server/shared/src/services/queue.service.ts:3]: Rollback everything except removing the finicity items

File: libs/server/shared/src/services/queue/bull-queue.ts
Comment (tmyracle) [inline libs/server/shared/src/services/queue/bull-queue.ts:1]: Don't remove these

File: libs/server/shared/src/services/queue/in-memory-queue.ts
Comment (tmyracle) [inline libs/server/shared/src/services/queue/in-memory-queue.ts:1]: dont remove these

**Comment by tmyracle** [comment]
Left a bunch of notes on things that shouldn't be deleted, also I'd expect to see migration files that are dropping the columns and enums that are removed in the schema. Is running `yarn prisma:migrate:dev` not generating any new migrations for these changes?

**Comment by lorenzopalaia** [comment]
@tmyracle Every listed issue from you should be fixed. For what concerns migrations, I'll check that.

**Comment by MichaelDeBoey** [comment]
@tmyracle @Shpigford #157 adds the necessary migrations + adds some extra removals of Finicity references as well

**Pull request closed without merging**